autotest/t081_test.py

```diff
@@ -0,0 +1,350 @@
+"""
+created matt_dumont 
+on: 15/02/22
+"""
+import flopy
+import numpy as np
+from ci_framework import FlopyTestSetup, base_test_dir
+import platform
+
+base_dir = base_test_dir(__file__, rel_path="temp", verbose=True)
+nrow = 3
+ncol = 4
+nlay = 2
+nper = 1
+l1_ibound = np.array([[[-1, -1, -1, -1],
+                       [-1, 1, 1, -1],
+                       [-1, -1, -1, -1]]])
+l2_ibound = np.ones((1, nrow, ncol))
+l2_ibound_alt = np.ones((1, nrow, ncol))
+l2_ibound_alt[0, 0, 0] = 0
+ibound = {
+    'mf1': np.concatenate((l1_ibound, l2_ibound), axis=0),  # constant heads around model on top row
+    'mf2': np.concatenate((l1_ibound, l2_ibound_alt), axis=0),  # constant heads around model on top row
+}
+laytype = {
+    'mf1': [0, 1],
+    'mf2': [0, 0]
+}
+hnoflow = -888
+hdry = -777
+top = np.zeros((1, nrow, ncol)) + 10
+bt1 = np.ones((1, nrow, ncol)) + 5
+bt2 = np.ones((1, nrow, ncol)) + 3
+botm = np.concatenate((bt1, bt2), axis=0)
+ipakcb = 740
+
+names = ['mf1', 'mf2']
+
+mf2005_exe = "mf2005"
```

Comment:
We typically set a run variable based on the presence of the required executable for a test.  

```
exe_names = {"mf2005": "mf2005", "mf6": "mf6", "mp7": "mp7"}
run = True
for key in exe_names.keys():
    v = flopy.which(exe_names[key])
    if v is None:
        run = False
        break
```

and use it later like this:

```
    # if models not run then there will be no output
    if not run:
        return
```

This way, if the executable is not present, which may be the case for some developers, then we still test flopy's ability to write files.  This might be why you added mfpymake to the requirements, but this would not be consistent with our current approach.

Reply:
Thanks for the note.  As per above, are these standard practices documented somewhere I missed?  I have implemented this (in next commit)

Reply:
These standard practices are not documented, but we agree that they should be.